modules/rpc/yarpc.go

```diff
@@ -49,6 +50,7 @@ type YARPCModule struct {
 	log       ulog.Log
 	stateMu   sync.RWMutex
 	isRunning bool
+	di        dig.Graph
```

Comment:
Short answer because of snowflakes and tests: it is much easier to write tests with custom graphs, especially when/if yarpc team will add an in memory transport.

Also, I was able to remove di from the struct. Wrote a long rant, why it is hard and then hit an eureka moment..